docker/uwsgi.ini

```diff
@@ -0,0 +1,6 @@
+[uwsgi]
+socket = 0.0.0.0:3031
+module = mbspotify
+callable = create_app()
+chdir = /code/
+processes = 20
```

Comment:
Is this the non-production version? If so, 20 is really high for non production. Maybe 5?